content/docs/user-guide/project-structure/internal-files.md

```diff
@@ -61,8 +61,8 @@ operation.
   dependencies and outputs, to allow safely running multiple DVC commands in
   parallel
 
-- `.dvc/tmp/exp<n>`: workspace copy number _n_ for
-  [parallel experiments](/doc/command-reference/exp/run#queueing-and-parallel-execution).
+- `.dvc/tmp/exp<n>`: _n_<sup>th</sup> workspace copy, used for temporary or
+  parallel <abbr>experiments</abbr> (see `dvc exp run`).
```

Comment:
Gm @pmrowla 👋 Is it `exp<n>` for `--temp` runs too? What does _n_ mean in those cases? (For parallel runs it's the job # I think.)

Reply:
The directory is temp dir runs, the path should look like `.dvc/tmp/exps/...`. The `<something>` is automatically generated by python and depends on the system/platform, it's just a unique temporary directory name, the actual implementation should not be documented.

The only thing that should be documented here is that for parallel or temporary experiment runs, `.dvc/tmp/exps/` will be used as the default location for temporary workspace directories (rather than the default system `$TMPDIR`).

Reply:
Why did you think I intended to document the exact implementation details? I'm confirming about `exp<n>` which I got from somewhere and seems strange for `--temp`. Trying to keep https://dvc.org/doc/user-guide/project-structure/internal-files up to date...

Updated 👍